Sources/Orbit/Components/InputField.swift

```diff
@@ -160,15 +195,64 @@ public extension InputField {
     }
 }
 
+private struct SecureTextField: UIViewRepresentable {
```

Comment:
Ideally this should be moved to `Support` folder, along with its own preview (similar to existing UIViewRepresentable wrappers that are there)